common/shlibs

```diff
@@ -4303,3 +4303,4 @@ libunicode_loader.so.0.4 libunicode-0.4.0_1
 force-stage.so.0.1 void-force-stage-0.1_1
 libliftoff.so.0 libliftoff-0.4.1_1
 libscfg.so libscfg-0.1.1_1
+libgnome-games-support-2.so.4 libgnome-games-support2-2.0.0_1
```

Comment:
Put this near libgnome-games-support-1.so.3

Reply:
fixed